bigdata/src/main/scala/org/w3/banana/bigdata/BigdataConfig.scala

```diff
@@ -0,0 +1,36 @@
+package org.w3.banana.bigdata
+
+import java.util.Properties
+
+
+/**
+ * Bigdata configuration, Rdf parameter is used to resolve it from module
+ * @tparam Rdf
+ */
+trait BigdataConfig[Rdf<:Bigdata]
+{
+  def journal:String
+  def basePrefix:String
+  def properties:Properties
+}
+
+case object DefaultBigdataConfig extends BigdataConfig[Bigdata]{
```

Comment:
I can remove "case" from it. Actually the only difference between case object and just object is that case objects can be serialized while regular objects cannot